haalcentraal/haalcentraal-brp/src/main/kotlin/com/ritense/haalcentraal/brp/connector/HaalCentraalBrpConnector.kt

```diff
@@ -14,35 +14,35 @@
  * limitations under the License.
  */
 
-package com.ritense.haalcentraal.connector
+package com.ritense.haalcentraal.brp.connector
 
 import com.ritense.connector.domain.Connector
 import com.ritense.connector.domain.ConnectorProperties
 import com.ritense.connector.domain.meta.ConnectorType
-import com.ritense.haalcentraal.client.HaalCentraalBRPClient
-import com.ritense.haalcentraal.domain.Person
-import com.ritense.haalcentraal.domain.Personen
-import com.ritense.haalcentraal.web.rest.request.GetPeopleRequest
+import com.ritense.haalcentraal.brp.client.HaalCentraalBrpClient
+import com.ritense.haalcentraal.brp.domain.Person
+import com.ritense.haalcentraal.brp.domain.Personen
+import com.ritense.haalcentraal.brp.web.rest.request.GetPeopleRequest
 import kotlinx.coroutines.runBlocking
 import java.security.InvalidParameterException
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 
-@ConnectorType(name = "HaalCentraal")
-class HaalCentraalBRPConnector(
-    private var haalCentraalBRPProperties: HaalCentraalBRPProperties,
-    private var haalCentraalBRPClient: HaalCentraalBRPClient
+@ConnectorType(name = "HaalCentraalBrp")
```

Comment:
Moet er een migratiescript komen om dit automatisch om te zetten in een bestaande database?

Reply:
oh never mind I see it already exists

Reply:
I see one it exists for the properties but not the type

Reply:
I did add them [here](https://github.com/valtimo-platform/valtimo-backend-libraries/blob/feature/33714_API_BRP_specificeren_in_HaalCentraal_code/haalcentraal/haalcentraal-brp/src/main/resources/config/liquibase/changelog/20220413-rename-connector-to-haalcentraal-brp.xml#L23-L25)

Reply:
ah sorry missed that one